common/json.c

```diff
@@ -65,6 +66,30 @@ bool json_to_u64(const char *buffer, const jsmntok_t *tok,
 	return true;
 }
 
+bool json_to_s64(const char *buffer, const jsmntok_t *tok, int64_t *num)
```

Comment:
s/`int64_t`/`s64`/ .  Any code file that uses `u32` has included `ccan/short_types/short_types.h`

Reply:
Done in fb8ab0aa.